### DISCUSSION TITLE
DTCyber Deadstart is hanging @ RECOVERING DP006 and "ERROR ON SYSTEM DEVICE"

### DISCUSSION THREAD

**Discussion by EarwainBanadar** [kej715/DtCyber#68] [Q&A] [answered]
Hello all,

I did a fresh pull of DtCyber from github and did a 32bit compile on Ubuntu.

A sudo node start will bring up the console display (as I'm used to by Tom Hunter's DtCyber as well as being a user on a Cyber 176 once).

Xterm is telling me "DtCyber started - deadstarting NOS 2.8.7", the Cyber Console emulation is telling me "RECOVERING DP 006" but "Error on System Device" below.

The system is in AUTO mode and the "DISK BUSY".

This seems not to be a normal DEADSTART status ...

A.

**Comment by wschaub** [comment, marked as answer]
Did you do a node install first before node start? it hasn't initialized anything yet if not. 
if you have already done a node install I would try ../dtcyber manual
select option O
select option P
D=YES
hit enter
NEXT.
INITIALIZE,AL,5,6.
GO.

**Reply by EarwainBanadar** [reply]
Will give this a try ..

**Reply by EarwainBanadar** [reply]
Ok, to read a README is sometimes helpful ;-)

After "install" and entering the time I'm getting the AB display :-)
